src/lib/querys.ts

```diff
@@ -107,3 +107,19 @@ export const personQuery = groq`
     lastName,
   }
 `;
+
+export const reactGroupQuery = groq`
+*[_type == 'reactGroup']{
```

Comment:
Filter by approved?

Reply:
Also, could you please do me a favor and rename this file to `queries.ts`? That `querys` is killing me...